examples/heat-source-mpi.py

```diff
@@ -1,3 +1,5 @@
+"""Demonstrate heat source."""
+
```

Comment:
I acknowledge that this change is needed to get your linter to shut up, but it is just adding noise to the AV change set.  Not sure how others feel about this, but I'd just back this out and let it be changed by the file author, if they desire.  (cc @majosm).

Reply:
Yeah, I think we only enforce pydocstyle for the `mirgecom/` directory.

Reply:
Removed: https://github.com/illinois-ceesd/mirgecom/pull/213/commits/8a50f22a38d468b311358214e4791b4cab9f47b5